odkx-src/xlsx-converter-reference.rst

```diff
@@ -482,6 +482,16 @@ Available :th:`setting_name` values that can be used:
     - | Optional
     - | Used with :th:`display.prompt.text.<language>`, or
       | other fiels to set other language options in the form.
+  * - showHeader
+    - | Optional
+    - | Used to display the header at the top of every page of the form.
+      | Shows the form title in a navigation bar with Back and Next buttons.
+      | The form header is visible by default.
+  * - showFooter
+    - | Optional
+    - | Used to display a navigation bar at the bottom of 
+      | every page of the form. Similar to the header with Back
```

Comment:
There is typically formatting applied so the back and next buttons look like buttons in the docs

Reply:
sure, let me check out how to add that